srcpkgs/gst-plugins-base1/template

```diff
@@ -1,42 +1,43 @@
 # Template file for 'gst-plugins-base1'
 pkgname=gst-plugins-base1
-version=1.16.2
-revision=2
+version=1.18.1
+revision=1
 wrksrc="${pkgname/1/}-${version}"
 build_style=meson
 build_helper="gir"
 configure_args="-Dtremor=disabled -Dexamples=disabled -Dgtk_doc=disabled
  -Dcdparanoia=$(vopt_if cdparanoia enabled disabled)
- -Dintrospection=$(vopt_if gir enabled disabled)
- -Dsndio=$(vopt_if sndio enabled disabled)"
+ -Dintrospection=$(vopt_if gir enabled disabled)"
 hostmakedepends="gettext pkg-config glib-devel orc"
 makedepends="gstreamer1-devel glib-devel libxml2-devel pango-devel
  cairo-devel liboil-devel alsa-lib-devel libXv-devel libXext-devel
  libvisual-devel libgudev-devel libtheora-devel libvorbis-devel
  libSM-devel orc-devel $(vopt_if cdparanoia libcdparanoia-devel)
- opus-devel MesaLib-devel $(vopt_if sndio sndio-devel) graphene-devel"
+ opus-devel MesaLib-devel graphene-devel"
 depends="orc>=0.4.18 gstreamer1>=${version}"
 short_desc="GStreamer Base Plug-ins (v1.x)"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.0-or-later"
 homepage="https://gstreamer.freedesktop.org"
 distfiles="${homepage}/src/${pkgname/1/}/${pkgname/1/}-${version}.tar.xz"
-checksum=b13e73e2fe74a4166552f9577c3dcb24bed077021b9c7fa600d910ec6987816a
+checksum=1ba654d7de30f7284b4c7071b32f881b609733ce02ab6d9d9ea29386a036c641
 
 # Package build options
-build_options="cdparanoia gir sndio"
+build_options="cdparanoia gir"
 build_options_default="cdparanoia gir"
 
-# this should not remain in this package in longer term
-# either upstream, separate, or remove
-desc_option_sndio="Enable sndio support (unsupported, known to be problematic)"
+do_check() {
+	: # Tests needs to be ran outside of chroot
+}
```

Comment:
why do they need to be outside chroot? maybe they need dbus or something?

Reply:
Starting to believe it may need itself more so than anything. Researching errors like it was giving test, people were saying you need gst-plugins-base.